documentation/assemblies/assembly-iot-guide.adoc

```diff
@@ -11,23 +11,10 @@ The IoT guide provides resources on how to set up and manage {ProductName} IoT f
 
 include::../modules/con-iot-concepts.adoc[leveloffset=+1]
```

Comment:
Suggest including as part of the IoT getting started assembly instead (at the beginning)